integration-tests/goss/centos7/goss-expected-q.yaml

```diff
@@ -55,38 +55,10 @@ command:
     stderr: []
     timeout: 10000
 dns:
-  CAA:dnstest.io:
```

Comment:
Could we comment these out with a TODO(ticket) to put them back in? Removing them sounds like it'll become more difficult to upstream later.

Reply:
If I put comments to this file, integration tests will fail.